### ISSUE TITLE
TextSecondary color in Blau to pass accessibility

### ISSUE THREAD

**Issue by yceballost** [Telefonica/mistica-design#1687]
### Discussed in https://github.com/Telefonica/mistica-design/discussions/1677

<div type='discussions-op-text'>

<sup>Originally posted by **DiegodeMiguel** February 26, 2024</sup>
### Goal

Accesibility for Blau callout don´t pass actually. (Textsecondary over backgroundContainerAlternative)

### Context

We have received feedback from the OB in Blau about the accesibility for the callout. Actually this fail: 
<img width="1233" alt="image" src="https://github.com/Telefonica/mistica-design/assets/106736588/aaed7c75-3f54-4084-b462-840caab9d630">
I copy here also the link from the tokens https://mistica-design.vercel.app/tokens-map/production/blau/color/backgroundContainerAlternative/
The OB tell us that we need to make it accesible and change the colour, they propose "If you change the grey text to 687173 the ratio is 4.5:1 and works"...but I don´t sure if we need to change the token colour about the "Secondarytext" or it´s better to unify with the other brands and change the background to put it into grey colour, because now Blau its the only one that its different:  
<img width="585" alt="image" src="https://github.com/Telefonica/mistica-design/assets/106736588/27de2570-4723-484d-853b-6cec38278390">
Thanks.

### Impact

This affect to all callouts. I attach a figma link as example https://www.figma.com/file/NPyjxLokVg6KLkdVrvkR7w/Blau_INVOICES?type=design&node-id=3958-14638&mode=design&t=pQAiNC4CAxvmybte-4

### Discussion Questions

Should we evolve this component or just find a new token colour?

### Do you have a visual proposal?

_No response_</div>
